packages/block-editor/src/components/rich-text/index.js

```diff
@@ -431,6 +431,10 @@ export function RichTextWrapper(
 				aria-multiline={ ! disableLineBreaks }
 				aria-readonly={ shouldDisableEditing }
 				{ ...props }
+				// Unset draggable for contentEditable elements because it will
+				// interfere with multi block selection when the contentEditable
+				// and draggable elements are the same element.
+				draggable={ undefined }
```

Comment:
Which draggable we're unsetting, the one set by the block wrapper I guess. Maybe it's a good thing to mention.

Reply:
Added a mention